elasticsearch/src/main/java/zipkin/dependencies/elasticsearch/ElasticsearchDependenciesJob.java

```diff
@@ -61,6 +61,9 @@ public static Builder builder() {
 
     Builder() {
       sparkProperties.put("spark.ui.enabled", "false");
+      // maybe support more spark properties?
+      sparkProperties.put("spark.hadoop.yarn.resourcemanager.address",
+              System.getProperty("spark.hadoop.yarn.resourcemanager.address", "0.0.0.0:8032"));
```

Comment:
where does 0.0.0.8032 come from? Is this a default? if so, we should add comment of the documentation page or github link describing it.

Reply:
it seems default to [${yarn.resourcemanager.hostname}:8032](https://hadoop.apache.org/docs/r2.7.4/hadoop-yarn/hadoop-yarn-common/yarn-default.xml) and `yarn.resourcemanager.hostname` is default to `0.0.0.0`.